### DISCUSSION TITLE
Primevue apollo template leftmenu toggle problem

### DISCUSSION THREAD

**Discussion by aligbayir** [primefaces/community#2058] [PrimeVue Templates]
Hello 
I am trying to toggle all left menu with searchbar but when I try filtered menus came collapsed how can i fix that ? is there any solution or example like that ?
![image (8)](https://github.com/primefaces/community/assets/78231112/63ce1cfb-7e3f-496a-9a8a-ad43a4a10948)
![image (7)](https://github.com/primefaces/community/assets/78231112/eea636a8-fd61-401e-b4ce-cad6d9296199)
![image (6)](https://github.com/primefaces/community/assets/78231112/2a42776b-7947-493d-95f4-7a9da28a7989)

**Comment by cagataycivici** [comment]
We do not have a built-in search bar for the menu and unable to support customizations like these on the template.
